site/src/sphinx/zookeeper.rst

```diff
@@ -0,0 +1,55 @@
+.. _`Apache ZooKeeper`: https://zookeeper.apache.org/
+.. _`com.linecorp.armeria.client.zookeeper`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/package-summary.html
+.. _`com.linecorp.armeria.server.zookeeper`: apidocs/index.html?com/linecorp/armeria/server/zookeeper/package-summary.html
+.. _`StoreType`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/StoreType.html
+
+Automatic service discovery with ZooKeeper
+===========================================
+You can put your Endpoints information into an zNode in `Apache ZooKeeper`_ Cluster, as a node value or as the child value of a node, like the following:
+
+| as node value:
+|         /myProductionEndpoints --> 192.168.1.10:8080:100,192.168.1.11:8080:100
+
+| as node children:
+|        /myProductionEndpoints
+|                  /192.168.1.10_8080 --> 192.168.1.10:8080:100
+|                  /192.168.1.11_8080 --> 192.168.1.11:8080:100
+
+As it is very simple, 192.168.1.10 and others IP strings are your servers' IP address, 8080 is the service port number, 100 is the server's weight.You can omit the server's weight as it is optional.
+
+The above endpoint groups can be retrieved using the corresponding `StoreType`_:
+
+.. code-block:: java
+
+     EndpointGroup myEndpointGroup = new ZooKeeperEndpointGroup(zkConnectionStr, zNodePath, sessionTimeout,storeType);
+     ..
+
+
+After creating an EndpointGroup, you can register it, and then use it in client service setup:
+
+.. code-block:: java
+
+     EndpointGroupRegistry.register("myProductionGroup", myEndpointGroup, WEIGHTED_ROUND_ROBIN);
+     HelloService.Iface helloClient = Clients.newClient("tbinary+http://group:myProductionGroup/hello",HelloService.Iface.class);
```

Comment:
- ,HelloService -> , HelloService
- Global comment: Try to generate the web site and make sure the code snippets do not have any horizontal scroll bars, for readability.